README.md

```diff
@@ -38,6 +38,30 @@ Open up [http://localhost:3000](http://localhost:3000) and you should be ready t
 
 If you're used to using the `vercel dev` command provided by [Vercel CLI](https://vercel.com/cli) instead, you can also use that, but it's not needed.
 
+### Stripe Webhook Forwarding
+
+If you wish to test the Stripe webhook flow, you must use the Stripe CLI to forward the webhook to your local environment.
+
+[Full instructions can be found on the Stripe documentation page.](https://docs.stripe.com/stripe-cli/overview#forward-events-to-your-local-webhook-endpoint)
+
+You must initialize the Stripe CLI with your Stripe account:
+
+```sh
+stripe login
+```
+
+Then run the following to start forwarding webhooks:
+
+```sh
+stripe --api-key <STRIPE API KEY> listen --forward-to http://localhost:3000/api/stripe/webhook
```

Comment:
Can you link to our private 1password repo with where to get the STRIPE_API_KEY?

Reply:
should we update the commend to ` ... --api-key {STRIPE_API_KEY} ...`

Reply:
sure. updated both.